### DISCUSSION TITLE
Creating a model - not working

### DISCUSSION THREAD

**Discussion by dinoflint** [TheLastBen/fast-stable-diffusion#2954] [General]
Hi Everyone,

I am new at this and this is my first go at this.

I have been following the instructions I received to install all this but something doesn't seem to be working and I can't work it out and nothing I have looked at seems to have an answer.

I am working on Gradient Notebook which has Stable Diffusion.

I have clicked on it and Run it but it just keeps loading and nothing seems to happen.

when I refresh the page it looks like the following:

![image](https://github.com/user-attachments/assets/08bd4aff-e72e-4d02-bd8d-e12b9de2199f)

I don't know if it has down loaded properly or if there is more to do but when I click on models nothing shows up even thought it says the model has downloaded.

![image](https://github.com/user-attachments/assets/be80560e-0e0b-4672-ae65-d718e030ead4)

When I click on models none show and I get the following error:

![image](https://github.com/user-attachments/assets/f6194a90-73c8-45eb-90cf-e67c32afe125)

![image](https://github.com/user-attachments/assets/e061adc9-cdfb-4ade-8479-781628a490b2)

![image](https://github.com/user-attachments/assets/155e577a-0670-4f07-9e98-60b5629a024a)

![image](https://github.com/user-attachments/assets/af565ee3-a48c-459b-ad53-101d11e4ea6e)

![image](https://github.com/user-attachments/assets/32c8d8c3-94eb-47f4-bc57-c0756245a720)

Any assistance will be greatly appreciated.

**Comment by TheLastBen** [comment]
you need to click on the url in the output
"Running on local URL"

**Reply by dinoflint** [reply]
Hi,
Thank you for your reply.
I am not able to find that.
I would be grateful if you can please let me know where I can find that.
I am new at this and trying to find my way.
Thank you & best regards,
Joe

**Comment by dinoflint** [comment]
Hi Ben,
I am new a this and would be grateful if you can assist with where I find the URL "Running on local URL"
Thank you and best regards,

    On Tuesday 19 November 2024 at 07:37:09 am AEDT, Ben ***@***.***> wrote:  
 
 


you need to click on the url in the output
"Running on local URL"
